1-exercises/D-for-in-loops/exercise-2.js

```diff
@@ -25,6 +25,11 @@ let shapesWithMoreThanFiveSides = [];
 
 // WRITE YOUR CODE BELOW THIS LINE
 
+for (key in shapes) {
+  let shapeName = key;
```

Comment:
Nice solution Helen, you can declare the variable in the for in loop. That way we wont need line 29.